### PR TITLE
Improve memory useage

### PR DESCRIPTION
**Pull request by Dreagonmon**
Use bytes to save memory

### CODE REVIEW

**Comment by JASchilz** [comment]
Acknowledged! I think this is a welcome change, and I'll did into it this
weekend.

**Comment by Dreagonmon** [comment]
This change may not improve memory useage at runtime, but it can shrink down the code size when compiling to .mpy file.
When you don`t freeze the code into ROM, it will be loaded into memory at runtime. This will use less memory when you using uQR in the following way:

```
pip install mpy_cross
python -m mpy_cross uQR.py
# copy uQR.mpy to micropython board`s file system
# import uQR
# ......
```

There is no need to compile the whole micropython firmware, and it is also possible to run uQR on ESP8266.  :p

File: test.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+# print(']')
+
+from uQR import QRCode
+
+if __name__ == "__main__":
+    qr = QRCode()
+    qr.add_data('uQR rocks!')
+    matrix = qr.get_matrix()
+    print(matrix)
+    pass
```
Comment (JASchilz) [inline test.py]: I really like the idea of adding a test, so I added one that could tell me right away whether or not to accept a merge request. 😄 

See https://github.com/JASchilz/uQR/pull/2/

I've run this test against your merge request and it passes! But now the files are in conflict. So go ahead and remove this from your branch to resolve the conflict.

**Review by JASchilz — COMMENTED** [review]
See review items.

As I said, this is a welcome merge request. Thank you!

File: test.py (hunk 1 above)
Comment (Dreagonmon) [reply test.py]: Resolved!

**Comment by JASchilz** [comment]
Thank you!

**Pull request merged**